src/verify.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+
+exercises = [
+    ("contracts/beginner", ["ex00", "ex01", "ex02", "ex03", "ex04"]),
```

Comment:
Can't we have a system where we retrieve all cairo files in each directory instead ?
That would be more flexible if we add more exercices IMO. We wouldnt have to change verify script

Reply:
It depends because we want to have the flexibility not to have ex00 ex01 etc but maybe storage00 storage01 memory00 memory01 for example.
Configuration over convention here would be our go-to strategy

Reply:
We want to have them checked in a specific order
example: vec00 and vec01 before hashmap00 and hashmap01
Which is not the alphabetic order or creation order or last modif order. It have to be hardcoded in a way somewhere